[ci skip] documentation for #36589 (#46698)
rails/rails: guides/source/active_record_migrations.md

* [`rename_index`][]
* [`rename_table`][]

[`change_table`][] is also reversible, as long as the block does not call irreversible operation

> which does not correspond to the operations listed above.

Author: `change_default` or `remove` may be reversible if proper option is given. Wording can be improved though.

> [`change_table`][] is also reversible, as long as the block does not call irreversible operation
> which does not correspond to the operations listed above.

Reviewer: Would it be more clear without the double negative?

```suggestion
[`change_table`][] is also reversible, as long as the block only calls
reversible operations like the ones listed above.
```
